data/json/professions.json

```diff
@@ -6141,8 +6141,8 @@
       "prof_auto_pistols_familiar"
     ],
     "skills": [
-      { "level": 7, "name": "gun" },
-      { "level": 7, "name": "rifle" },
+      { "level": 5, "name": "gun" },
+      { "level": 5, "name": "rifle" },
```

Comment:
Is this intentional? Seems very unrelated

Reply:
@Procyonae definitely isn't, and I definitely didn't touch this specific file in any way. As I mentioned something went wrong with Github (possibly because of me).